src/ui/CircularElement.cpp

```diff
@@ -0,0 +1,46 @@
+#include "CircularElement.h"
+
+CircularElement::CircularElement(Adafruit_GFX& display) :
+    VisualElement(display) { }
+
+CircularElement::~CircularElement() { }
+
+int16_t CircularElement::getWidth() {
+    return _radius * 2;
+}
+
+int16_t CircularElement::getHeight() {
+    return _radius * 2;;
```

Comment:
;;

Reply:
oh thanks good catch

compiler didn't seem to mind, I guess it probably ignored it

Reply:
https://stackoverflow.com/questions/36408249/why-putting-2-semicolon-is-not-an-error

Reply:
It makes sense I suppose.  I never realized it wouldn't even come up as a warning though

Reply:
lol, guess an empty statement is still a statement